src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/MemoryPoolViewBufferScope.cs

```diff
@@ -60,7 +65,7 @@ public ViewBufferValue[] GetSegment()
 
             try
             {
-                segment = _viewBufferPool.Rent(SegmentSize);
+                segment = _viewBufferPool.Rent(Math.Max(size, MinimumSize));
```

Comment:
Given all the callers are also `.Internal`, do we really need to worry about incorrect calls to this method?